docs/features/software-templates/authorizing-scaffolder-tasks-parameters-steps-and-actions.md

```diff
@@ -1,10 +1,10 @@
 ---
-id: authorizing-parameters-steps-and-actions
-title: 'Authorizing parameters, steps and actions'
-description: How to authorize part of a template
+id: authorizing-scaffolder-tasks-parameters-steps-and-actions
```

Comment:
```suggestion
id: authorizing-scaffolder-template-details
```
If we are going to change this it might make sense to have something more generic?

Reply:
We should also rename the file to `authorizing-scaffolder-template-details` and also add a redirect for those with links to the current version. You would add the redirect here: https://github.com/backstage/backstage/blob/c018a79584ac25dfd386ebff621fac406a980a05/microsite/docusaurus.config.ts#L125